<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InAndOutDetailPage</class>
 <widget class="QWidget" name="InAndOutDetailPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="backgroundwidget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7" rowstretch="15,1,75">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QWidget" name="Bottomwidget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4" rowstretch="2,30">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="widget_subwidget" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>15</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>15</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,2,2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>20</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <property name="bottomMargin">
               <number>20</number>
              </property>
              <item>
               <widget class="QLabel" name="label_singelnumrank_subtitle">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;微软雅黑&quot;;</string>
                </property>
                <property name="text">
                 <string>单笔支出排行</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sortBynum_Button">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;微软雅黑&quot;;
background-color: rgba(200, 200, 200,100);
</string>
                </property>
                <property name="text">
                 <string>按金额</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sortBytime_Button">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;微软雅黑&quot;;
</string>
                </property>
                <property name="text">
                 <string>按时间</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="DetialTableview" name="detailview">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_6" rowstretch="2,7">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout" columnstretch="2,15,2">
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="rightMargin">
           <number>15</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="returnButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;
border-image: url(:/public/image/returnarrow.jpg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_Title_Date">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
font: 23pt &quot;微软雅黑&quot;;
background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>2023年01月</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">
background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="5,3,7,5">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="2" column="0">
           <widget class="QLabel" name="label_num">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
font: 38pt &quot;微软雅黑&quot;;
background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>600.00</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_InorOut_Type">
            <property name="styleSheet">
             <string notr="true">color: rgb(141,141,141);
font: 15pt &quot;微软雅黑&quot;;
background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>本月总支出</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(200, 200, 200,70);</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DetialTableview</class>
   <extends>QTableView</extends>
   <header>detialtableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
